src/api/Feed.js

```diff
@@ -165,27 +166,38 @@ class Feed extends Base {
     updateAnnotation = (
         file: BoxItem,
         annotationId: string,
-        text: string,
+        text?: string,
+        status?: FeedItemStatus,
         permissions: AnnotationPermission,
         successCallback: (annotation: Annotation) => void,
         errorCallback: ErrorCallback,
     ): void => {
         if (!file.id) {
             throw getBadItemError();
         }
+        if (!text && !status) {
```

Comment:
What's the user experience in this case?

Reply:
Currently there is no user experience. This situation would be abnormal just as `!file.id` above. It only throws an error. We could update the UI with running `this.updateCommentErrorCallback` for both of the cases just as it's done when annotationsAPI calls errorCallback. What do you think?